scaffolding-chef-infra/tests/test-default.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+#/ Usage: test.sh <pkg_ident>
+#/
+#/ Example: test.sh chef/scaffolding-chef-infra/1.2.0/20181108151533
+#/
+
+set -euo pipefail
+
+if [[ -z "${1:-}" ]]; then
+  grep '^#/' < "${0}" | cut -c4-
+	exit 1
+fi
+
+TEST_PKG_IDENT="${1}"
+export TEST_PKG_IDENT
+TEST_PKG_NAME="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f2)"
+export TEST_PKG_NAME
+
+source "$(dirname "${0}")/${TEST_PKG_NAME}/habitat/plan.sh"
+export pkg_svc_path
+
+hab pkg install core/bats --binlink
+hab pkg install "${TEST_PKG_IDENT}"
+
+SUP_WAIT_SECONDS=1
+
+hab sup term
+sleep "${SUP_WAIT_SECONDS}"
+echo "--- :habicat: Starting the supervisor"
+if [ ! -f /hab/sup/default/sup.log ]; then
```

Comment:
I believe lines 30-34 are leftovers from previous test attempts? Should these be removed?